.github/workflows/check-test-release.yml

```
@@ -1,15 +1,22 @@
 name: Check, test and release
 on:
   push:
-  pull_request:
+  pull_request_target:
   schedule:
   - cron: '0 7 * * 1'  # M H d m w (Mondays at 7:00)
 
 env:
   MLEM_TESTS: "true"
 
 jobs:
+  authorize:
+    environment:
+      ${{ (github.event_name == 'pull_request_target' &&
+      github.event.pull_request.head.repo.full_name != github.repository) &&
+      'external' || 'internal' }}
+    runs-on: ubuntu-latest
   check:
+    needs: authorize
     if: startsWith(github.ref, 'refs/tags') || github.event_name == 'pull_request' || github.event_name == 'schedule' || github.repository_owner != 'iterative'
```

```suggestion
    if: startsWith(github.ref, 'refs/tags') || github.event_name == 'pull_request_target' || github.event_name == 'schedule' || github.repository_owner != 'iterative'
```

Forgot to update everywhere?

---

Resolved with https://github.com/iterative/mlem/pull/318/commits/af101ae03931559874c7a129a7c28f1b4a107939